activeadmin.gemspec

```diff
@@ -17,16 +17,16 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = %w[CHANGELOG.md CODE_OF_CONDUCT.md CONTRIBUTING.md README.md]
 
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.5'
 
   s.add_dependency 'arbre', '~> 1.2', '>= 1.2.1'
-  s.add_dependency 'formtastic', '~> 3.1'
-  s.add_dependency 'formtastic_i18n', '~> 0.4'
-  s.add_dependency 'inherited_resources', '~> 1.7'
-  s.add_dependency 'jquery-rails', '~> 4.2'
-  s.add_dependency 'kaminari', '~> 1.0', '>= 1.0.1'
+  s.add_dependency 'formtastic', '>= 3.1'
+  s.add_dependency 'formtastic_i18n', '>= 0.4'
+  s.add_dependency 'inherited_resources', '>= 1.7'
+  s.add_dependency 'jquery-rails', '>= 1.0'
+  s.add_dependency 'kaminari', '~> 1.0', '>= 1.1.1'
   s.add_dependency 'railties', '>= 5.2', '< 6.1'
-  s.add_dependency 'ransack', '~> 2.1', '>= 2.1.1'
-  s.add_dependency 'sassc-rails', '~> 2.1'
+  s.add_dependency 'ransack', '>= 2.1', '<= 2.3.2'
+  s.add_dependency 'sassc-rails', '>= 2.1', '<= 2.1.2'
```

Comment:
Same thing as in the case of `ransack`, we don't want to restrict upper bounds like that.

Reply:
I have to think about this one, perhaps: 
`  s.add_dependency 'sassc-rails', '>= 2.1'`

Reply:
What's the problem with the current specification?